pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.sh

```diff
@@ -102,21 +102,32 @@ while [ "$#" -gt 0 ]; do
       --use-substitutes|--substitute-on-destination|-s)
         copyFlags+=("-s")
         ;;
-      -I|--max-jobs|-j|--cores|--builders|--log-format)
+      --builders)
         j="$1"; shift 1
         extraBuildFlags+=("$i" "$j")
         ;;
-      --accept-flake-config|-j*|--quiet|--print-build-logs|-L|--no-build-output|-Q| --show-trace|--keep-going|-k|--keep-failed|-K|--fallback|--refresh|--repair|--impure|--offline|--no-net)
+      --I|--max-jobs|-j|--cores|--log-format)
```

Comment:
You broke `-I` by changing it to `--I`.

- #348886

Reply:
Ouch, thank you for finding